web/html/src/manager/state/recurring-states-list.js

```diff
@@ -116,25 +62,86 @@ class RecurringStatesList extends React.Component {
-                                    <th className="text-center">{t("Frequency")}</th>
-                                    {targetType}
-                                    <th className="text-right">{t("Actions")}</th>
-                                </tr>
-                                </thead>
-                                {this.tableBody()}
-                            </table>
-                            <DeleteDialog
-                                id="delete-modal"
-                                title={t("Delete Recurring State Schedule")}
-                                content={<span>{t("Are you sure you want to delete the selected item?")}</span>}
-                                onConfirm={() => this.props.onDelete(this.state.itemToDelete)}
-                                onClosePopUp={() => this.selectToDelete(null)}
+                        <div>
+                            <Table
+                                data={this.props.data}
+                                identifier={action => action.recurringActionId}
+                                //initialSortColumnKey="scheduleName"
+                                initialItemsPerPage={this.props.disableCreate ? userPrefPageSize : 0}
+                                emptyText={t("No schedules created." + (this.props.disableCreate ? "" : " Use Create to add a schedule."))}
+                            >
+                                <Column
+                                    columnKey="active"
+                                    header={t('Active')}
+                                    cell={(row, criteria) => {return (
+                                        <Toggler
+                                            value={row.active}
+                                            className="btn"
+                                            handler={() => this.props.onToggleActive(row)}
+                                        />
+                                    );
+                                    }}
+                                />
+                                <Column
+                                    columnClass="text-center"
+                                    headerClass="text-center"
+                                    columnKey="scheduleName"
+                                    header={t('Schedule Name')}
+                                    cell={(row, criteria) => row.scheduleName}
+                                />
+                                <Column
+                                    columnClass="text-center"
+                                    headerClass="text-center"
+                                    columnKey="frequency"
+                                    header={t('Frequency')}
+                                    cell={(row, criteria) => row.cron}
+                                />
+                                { this.props.disableCreate ?
```

Comment:
Maybe out of scope but, what was the reason for having this column optional? Can't we keep the same number of column, and maybe empty or grayed out content?

Reply:
I just thought it would be redundant information in those cases, since you already know that e.g. the actions you look at are minion actions if you have selected a system.

But I am not opposed to displaying that unconditionally, if you think it's the better aproach.

Reply:
Yeah, I see your POV and it makes sense generally speaking, but in the end it does not take a lot of space and it does not hijack the focus of it so, I'd say better to keep the consistency (which we lack a lot of cross the entire product) and keep the column always as they are. :wink: